### PR TITLE
Add support of `waiting` status from `/from_url/status` endpoint.

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

**Comment by bautrukevich** [comment]
[UploadClient: Добавить статус waiting для fromUrl](https://trello.com/c/iooKO3R2/499-uploadclient-%D0%B4%D0%BE%D0%B1%D0%B0%D0%B2%D0%B8%D1%82%D1%8C-%D1%81%D1%82%D0%B0%D1%82%D1%83%D1%81-waiting-%D0%B4%D0%BB%D1%8F-fromurl)

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## Added
+
+* Support of `waiting` status from `/from_url/status` endpoint.
```
Comment (nd0ut) [inline CHANGELOG.md]:
```suggestion
* Support of `waiting` status from `/from_url/status/` endpoint.
```

File: src/fileFrom/UploadFromUrl.ts [hunk 2]
```diff
@@ -53,10 +54,29 @@ export class UploadFromUrl extends UploadFrom {
   }
 
   private handleFromUrlStatusResponse = (token: string, response: FromUrlStatusResponse) => {
+    this.isFileUploadedFromUrlPolling = checkFileIsUploadedFromUrl({
```
Comment (homm) [inline src/fileFrom/UploadFromUrl.ts]: Further refactoring is required. There are two places, where response is checked: handleFromUrlStatusResponse and checkFileIsUploadedFromUrl. For me, pooler should be created unconditionally and make decisions about responses.

File: src/fileFrom/UploadFromUrl.ts [hunk 3]
```diff
@@ -53,10 +54,29 @@ export class UploadFromUrl extends UploadFrom {
...
+      token,
+      timeout: 1000,
```
Comment (homm) [inline src/fileFrom/UploadFromUrl.ts]: This is too few. Should be ten seconds, at least.

File: src/fileFrom/UploadFromUrl.ts (hunk 2 above)
Comment (bautrukevich) [reply src/fileFrom/UploadFromUrl.ts]: Yes, you are right.

File: src/fileFrom/UploadFromUrl.ts (hunk 3 above)
Comment (bautrukevich) [reply src/fileFrom/UploadFromUrl.ts]: I agree

**Pull request merged**